tfhe/docs/getting_started/benchmarks/README.md

```diff
@@ -12,4 +12,8 @@ make print_doc_bench_parameters
 
 ### Operation time over FheUint 64
 
+{% hint style="info" %}
+All CPU benchmarks in the Table below were launched on an `AWS hpc7a.96xlarge` instance equipped with two 48-core `AMD EPYC 9R14 CPU @ 2.60GHz` and 740GB of RAM, while all GPU benchmarks were launched on 1xH100 GPU, and rely on the multithreaded PBS algorithm.
```

Comment:
isn't it 96 cores and 192 threads ?

Reply:
It's 96 cores indeed, it's clearer to write "a 96-core" than "two 48-core" so I changed it everywhere.